regcomp.c

```diff
@@ -23741,6 +23744,36 @@ S_parse_uniprop_string(pTHX_
                     packWARN(WARN_EXPERIMENTAL__UNIPROP_WILDCARDS),
                     "The Unicode property wildcards feature is experimental");
 
+                if (special_property) {
+                    const char * error_msg;
+                    const char * revised_name = name + i;
+                    Size_t revised_name_len = name_len - (i + 1 + escaped);
+                    HV * table;
+
+
+                    /* Currently, the only 'special_property' is name, which we
+                     * lookup in _charnames.pm */
+
+                    table = load_charnames(newSVpvs("placeholder"),
```

Comment:
why do we want `table`? We don't appear to use it for anything.

Reply:
I think this was left over; I'll get rid of it